src/vmm/src/devices/virtio/iov_deque.rs

```diff
@@ -77,13 +77,11 @@ pub enum IovDequeError {
 //    pub iov_len: ::size_t,
 // }
 // ```
-//
-// This value must be a multiple of 256 because this is the maximum number of `iovec` can fit into
-// 1 memory page: 256 * sizeof(iovec) == 4096 == HOST_PAGE_SIZE. IovDeque only operates with
-// `HOST_PAGE_SIZE` granularity.
+
 #[derive(Debug)]
 pub struct IovDeque<const L: u16> {
     pub iov: *mut libc::iovec,
+    pub bytes: u32,
```

Comment:
I suppose this should be mentioned in the changelog.

Reply:
added a note in the Fixed section